test/functionalTest/cases/1179_options_test/GET_v2_ent_attr_val_options_text.test

```diff
@@ -42,49 +42,56 @@ echo "====================="
 payload='{
   "id": "E1",
   "type": "T1",
-  "a_string":"some value",
```

Comment:
Now that I had to modifiy this file, I fixed the indentations also. It was ugly ...

The real change is that **?options=text** is removed and that the response is in JSON for compound values